.rodare.json

```
@@ -0,0 +1,42 @@
+{
+    "creators": [
+	{
+            "name": "Eckert, Carlchristian",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden"
+        },
+        {
+            "name": "Widera, René",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid": "0000-0003-1642-0459"
+        },
+        {
+            "name": "Huebl, Axel",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid": "0000-0003-1943-7141"
+        },
+        {
+            "name": "Gruber, Bernhard Manfred",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, CASUS, CERN",
+            "orcid": "0000-0001-7848-1690"
+        }
+    ],
+    "contributors": [
+        {
+            "name": "Bastrakov, Sergei",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid": "0000-0003-3396-6154"
+        },
+        {
+            "name": "Worpitz, Benjamin"
+        },
+        {
+            "name": "Grund, Alexander"
```

If you like you can complete this with:

```suggestion
            "name": "Grund, Alexander"
            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
            "orcid": "0000-0002-7196-8452"
```